keanu-project/src/main/java/io/improbable/keanu/plating/Plates.java

```diff
@@ -1,9 +1,13 @@
 package io.improbable.keanu.plating;
 
 import java.util.ArrayList;
+import java.util.Iterator;
 import java.util.List;
 
-public class Plates {
+import org.jetbrains.annotations.NotNull;
+
+public class Plates implements Iterable<Plate> {
```

Comment:
I've always thought `Plates` should have better name to avoid `Plate` confusion. Any suggestions?

Reply:
Personally, I like `Plates`